docker/lb/Dockerfile.haproxy

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+FROM ubuntu:bionic
+
+RUN apt update \
+ && apt install -y -q \
+    haproxy
+
+COPY haproxy.cfg .
+RUN echo "$(cat haproxy.cfg)" >> /etc/haproxy/haproxy.cfg
```

Comment:
Can't we directly execute `COPY haproxy.cfg /etc/haproxy/haproxy.cfg`

Reply:
/etc/haproxy/haproxy.cfg has some contents, which I am not changing. I am just appending some extra setting for our purpose.

Reply:
ok. Please add a comment "Append front and back end load balancer configurations to existing file in /etc/haproxy/haproxy.cfg"

Reply:
Done